### DISCUSSION TITLE
UBlock Origin is looping MFA for QuickBooks Online

### DISCUSSION THREAD

**Discussion by dealingwithdigits** [uBlockOrigin/uAssets#13310] [Q&A]
Is there a fix that can be implemented for UBlock Origin looping Multi-factor Authentication (MFA) for QuickBooks Online https://qbo.intuit.com ?

We are being forced to redo MFA every 1 hour after QuickBooks Online times out or after closing the browser. 

The only way to circumvent this is to disable UBlock Origin.

Is there a script that can be blocked/unblocked to fix this?

**Comment by jsteave** [comment]
Is it possible[ ](https://www.theamericanschoolofprotocol.com/)to figure out what script we need to unblock in order to stop uBlock Origin from requiring MFA every time QuickBooks Online times out or the browser is closed?

If we disable the uBlock Origin extension, MFA stops looping for QuickBooks Online and is only required once -- like it is supposed to function -- but we want to use uBlock Origin.

Our IP address is static and we do not clear cookies.

**Reply by mapx-** [reply]
- open the logger
- repeat the steps to get the issue
- go back into the logger and then copy and paste here all blocked requests

**Comment by jsteave** [comment]
@[mapx-](https://www.dreamgarage.me/) thank you so much for your suggestion
